va/va.go

```diff
@@ -725,6 +768,57 @@ func (va *ValidationAuthorityImpl) validateChallenge(ctx context.Context, identi
 	return nil, probs.Malformed(fmt.Sprintf("invalid challenge type %s", challenge.Type))
 }
 
+func (va *ValidationAuthorityImpl) performRemoteValidation(ctx context.Context, domain string, challenge core.Challenge, authz core.Authorization, result chan *probs.ProblemDetails) {
```

Comment:
This just duplicates the signature of `PerformRemoteValidation` which uses the `domain` arg it is passed numerous times. That said we should probably file a ticket for that since you're right that we could just be using `authz.Identifier.Value`.